backend/database/models/attend.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+const { Model } = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class Attend extends Model {
+    static associations(models) {
+      this.eventAssociation = this.belongsTo(models.Booking, {
+        foreignKey: 'eventId',
+        as: 'booking', 
+        foreignKey: 'userId',
+        as: 'user'
+      });
+      this.eventAssociation = this.belongsTo(models.Booking, {
+        foreignKey: 'eventId',
+        as: 'booking'
+      });
+      this.userAssociation = this.belongsTo(models.Dependent, {
+        foreignKey: 'dependantId',
+        as: 'dependent'
+      });
+    }
+  };
+  Attend.init({
+    eventId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+      references: {
+        model: 'event',
+        key: 'id'
+      }
+    },
+    dependantId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+      references: {
+        model: 'dependent',
+        key: 'id'
+      }
+    },
+    userId: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        primaryKey: true,
+        references: {
+        model: 'user',
+        key: 'id'
+      }
+
+    }
+    // still need to create the migration for attend. 
```

Comment:
One suggestion I can give you for better code commenting, it is optional of course, is to use the following syntax:

Just write /***/ and it will directly give you a large space of comment and you can mention yourself using @elmehdielboustani .

<img width="354" alt="image" src="https://user-images.githubusercontent.com/75309623/204271837-cc18a148-807b-4194-8653-84709b6fed4c.png">

It is a good format of code commenting plus we know which of us wrote that comment and if is planned to be worked on.